packages/@aws-cdk/aws-route53-targets/test/integ.globalaccelerator-alias-target.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+import * as globalaccelerator from '@aws-cdk/aws-globalaccelerator';
+import * as route53 from '@aws-cdk/aws-route53';
+import * as cdk from '@aws-cdk/core';
+import * as targets from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-globalaccelerator-integ');
+
+let accelerator = new globalaccelerator.Accelerator(stack, 'Accelerator', {
+  acceleratorName: `${stack.stackName}`,
+  enabled: true,
+});
+
+const zone = new route53.PublicHostedZone(stack, 'HostedZone', { zoneName: 'test.public' });
+
+new route53.ARecord(stack, 'LocalGlobalAcceleratorAlias', {
+  comment: 'Alias to the locally created Global Accelerator',
+  target: route53.RecordTarget.fromAlias(new targets.GlobalAcceleratorTarget(accelerator)),
+  recordName: 'test-local',
+  zone,
+});
+
+new route53.ARecord(stack, 'ExistingGlobalAcceleratorAlias', {
```

Comment:
Either way you go above (option 2 or 3), I suspect we can just have a single instance here in the integ test. I trust our unit tests to ensure the two forms are equivalent, so deploying a single record should be sufficient to test that the generated CloudFormation is valid.

Reply:
I've left both resources in the integ test. I took a look at the CloudFormation, and the two resources are substantively different.
The IAccelerator version of the class winds up creating a GetAtt ref, whereas the string version creates a string literal AliasTarget DNSName. 
Since the resources are so different I think it's worth generating both so that we can prove each case in the expected json.

Does the build actually create the resources in a real account as part of the test? If speed is the concern, I can say I did run the build on my AWS account and the two aliases are created in parallel and don't take any extra time to deploy.

Reply:
Fair point, I've approved as-is.

> Does the build actually create the resources in a real account as part of the test?

The process is that the real-world deployment should be done to create the initial expected file, as well as to update it any time the expected result has changed; this means that the expected output always refers to a deployable template. The build just checks that the test still produces what is expected.